src/coreclr/jit/gentree.cpp

```diff
@@ -1840,6 +1842,39 @@ void CallArgs::Remove(CallArg* arg)
     assert(!"Did not find arg to remove in CallArgs::Remove");
 }
 
+#if TARGET_WASM
+void CallArgs::RemoveAfter(CallArg* arg)
+{
+    // if arg == nullptr, we are removing the first arg
+    if(arg == nullptr)
+    {
+        assert(m_head != nullptr);
+
+        m_head = m_head->GetNext();
+        return;
+    }
+
+    assert(arg->GetNext() != nullptr);
+    
+    arg->SetNext(arg->GetNext()->GetNext());
+}
+
+void CallArgs::MoveLateToEarly()
```

Comment:
Ditto: comment header.

Reply:
Do we need something in these comments to say it applies to LLVM lowering?

Reply:
No need I think. The `#ifdef` is self-explanatory and the functions are general-purpose.